### ISSUE TITLE
Fix RemovedInDjango20Warning: Importing from django.core.urlresolvers is deprecated in favor of django.urls.

### ISSUE THREAD

**Issue by lorddaedra** [aljosa/django-tinymce#192]
```
➜  concord docker exec concord-test /usr/local/bin/python3 -u -Wall /usr/src/app/manage.py check
/usr/local/lib/python3.5/site-packages/tinymce/widgets.py:16: RemovedInDjango20Warning: Importing from django.core.urlresolvers is deprecated in favor of django.urls.
  from django.core.urlresolvers import reverse
```

**Comment by claudep** [comment]
Issue addressed in PR #188

**Comment by claudep** [comment]
Fix committed in e9e5752840
